src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml

```diff
@@ -20,10 +20,7 @@
         <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
             <li><h6 class="dropdown-header">@T["Signed in as"]<span class="ms-1 font-weight-bold">@User.Identity.Name</span></h6></li>
             <li><hr class="dropdown-divider" /></li>
-            @*
-            TODO: Use CommonPermissions.EditOwnUser once 1.6.0 is released
-            *@
-            @if (await AuthorizationService.AuthorizeAsync(User, CommonPermissions.ManageUsers))
+            @if (await AuthorizationService.AuthorizeAsync(User, CommonPermissions.EditOwnUser))
```

Comment:
The build failed, we need to upgrade to `1.6.0`, I will update in another PR then I will merge this